resources/views/layouts/app.blade.php

```diff
@@ -0,0 +1,27 @@
+@extends('layouts/html')
```

Comment:
@abm-devmonk-purple Thoughts on using the more traditional blade "." syntax? `layouts.html`, etc.?

Reply:
Done in 23e307b0d6aa1a5e4c65f239610e3b8df43a45a9